### PR TITLE
T29698

### PR DESCRIPTION
**Pull request by GeorgesStavracas**
> Money, it's a crime
> Share it fairly but don't take a slice of my pie
> Money, so they say
> Is the root of all evil today
> But if you ask for a raise it's no surprise that they're
> Giving none away, away, away

This pull request is a courtesy of Nina the Dogga

nina no pc

https://phabricator.endlessm.com/T29698

### CODE REVIEW

File: js/misc/paygManager.js [hunk 1]
```diff
@@ -77,6 +77,7 @@ var PaygManager = GObject.registerClass({
         'enabled-changed': { param_types: [GObject.TYPE_BOOLEAN] },
         'expiry-time-changed': { param_types: [GObject.TYPE_INT64] },
         'initialized': {},
+        'rate-limit-end-time-changed':  { param_types: [GObject.TYPE_INT64] },
```
Comment (mwleeds) [inline js/misc/paygManager.js:80]: Should this be `TYPE_UINT64`?

Comment (mwleeds) [reply js/misc/paygManager.js:80]: And if it should, I guess `expiry-time-changed` should be too

**Comment by mwleeds** [comment]
LGTM other than the one question

File: js/misc/paygManager.js (hunk 1 above)
Comment (GeorgesStavracas) [reply js/misc/paygManager.js]: That's right, good catch!

**Pull request merged**